n26/designsystem/assets/app.css

```
@@ -64,6 +64,13 @@
  * Naming the full scale here costs a few KB and makes any shade legitimate. */
 @source inline("bg-{red,orange,amber,yellow,lime,green,emerald,teal,cyan,sky,blue,indigo,violet,purple,fuchsia,pink,rose,slate,gray,zinc,neutral,stone}-{50,100,200,300,400,500,600,700,800,900,950}");
 
+/* The mark drawn beside a tab whose panel has something waiting in it. Its
+ * markup is built in a view rather than a template, because the tab strip is
+ * assembled in the browser and takes its label decoration as a string — and a
+ * class named only in Python is a class the scan above never sees, so the
+ * element renders at nothing by nothing. */
+@source inline("inline-block size-2 rounded-full bg-accent align-middle");
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)